packages/runtime/test/consumption/iqlQuery.test.ts

```diff
@@ -320,20 +313,13 @@ describe("IQL Query", () => {
         expect(triggeredEvent!.data.newStatus).toBe(LocalRequestStatus.Completed);
     });
 
-    test("sender: sync Message with Response", async () => {
-        const result = await syncUntilHasMessages(sTransportServices);
-
-        expect(result).toHaveLength(1);
-
-        sResponseMessage = result[0];
-    });
-
-    test("sender: complete the outgoing Request with Response from Message", async () => {
+    test("sender: sync Message with Response and complete the outgoing Request with Response from Message", async () => {
         let triggeredEvent: OutgoingRequestStatusChangedEvent | undefined;
         sEventBus.subscribeOnce(OutgoingRequestStatusChangedEvent, (event) => {
             triggeredEvent = event;
         });
 
+        sResponseMessage = await syncUntilHasMessageWithResponse(sTransportServices, sLocalRequest.id);
```

Comment:
This `sReponseMessage` should be a local variable (as well as a lot of others which are defined in the header of the "IQL Query" test suite), which is part of a bigger problem: the tests share state, which is not something we want, as that makes it impossible to run and debug a single test in isolation.

Should be rectified either in this PR or a separate one.

Reply:
I'm still uncertain about if the tests run in parallel or not. If some of these tests are in fact run in parallel sharing state between them would explain any occasional test failures.

Reply:
@slavistan tests does never run in parallel, that's only the case for test suites.

Reply:
I created a separate backlog item for these coupled tests.